app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -111,5 +114,25 @@ def offer_store
     def action
       'back' if !!params[:back]
     end
+
+    def location_key
+      if @application_choice.different_offer?
+        'Preferred location'
+      else
+        text = 'not ' if @application_choice.school_placement_auto_selected?
+        "Preferred location (#{text}selected by candidate)"
+      end
+    end
+
+    def summary_list_rows
+      @summary_list_rows = [
+        { key: 'Full name', value: @application_choice.application_form.full_name },
+        { key: 'Course', value: @application_choice.course.name_and_code },
+        { key: 'Starting', value: @application_choice.course.recruitment_cycle_year },
+      ]
+      if @application_choice.different_offer?
+        @summary_list_rows << { key: location_key, value: @application_choice.site.name }
+      end
```

Comment:
This feels like a lot of presentation logic for a controller. How do you feel about moving this to a helper?